flink-connectors/flink-connector-aws/src/main/java/org/apache/flink/streaming/connectors/kinesis/util/AwsV2Util.java

```diff
@@ -130,7 +148,10 @@ public static SdkAsyncHttpClient createHttpClient(
             httpClientBuilder.connectionTimeToLive(Duration.ofMillis(config.getConnectionTTL()));
         }
 
-        return httpClientBuilder.build();
+        return httpClientBuilder.buildWithDefaults(
+                AttributeMap.builder()
+                        .put(SdkHttpConfigurationOption.TRUST_ALL_CERTIFICATES, trustAllCerts)
+                        .build());
```

Comment:
What prompted you to add this? Is it required for Kinesalite?

Reply:
This is required by Kinesalite I'm afraid.